helical/models/geneformer/geneformer_config.py

```diff
@@ -34,30 +34,35 @@ def __init__(
             emb_layer: int = -1,
             emb_mode: Literal["cls", "cell", "gene"] = "cell",
             device: Literal["cpu", "cuda"] = "cpu",
-            accelerator: Optional[bool] = False
+            accelerator: Optional[bool] = False,
             ):
         
         # model specific parameters
         self.model_map = {
             "gf-12L-95M-i4096": {
                 'input_size': 4096,
                 'special_token': True,
+                'embsize': 512,
```

Comment:
Do you need the embedding size ? Can't we infer that from the loaded model at a given moment? Once the base model is loaded, it should be visible what the size is of the embedding? (This comment is valid for all the sizes below)

Reply:
- It was just an easier way to make sure embedding sizes were correct
- I think this could be inferred from the base model. I never looked into that